src/testE2E/codecatalyst/client.test.ts

```diff
@@ -86,7 +86,7 @@ let projectName: CodeCatalystProject['name']
  *     integ tests, but using the ssh hostname that we get from
  *     {@link prepareDevEnvConnection}.
  */
-describe.skip('Test how this codebase uses the CodeCatalyst API', function () {
+describe('Test how this codebase uses the CodeCatalyst API', function () {
```

Comment:
if the throttling error still continues we may need to keep this skip()

Reply:
AFAIK this only runs with the `/runIntegrationTests` command right? And they aren't required to merge. Does it matter if it stays skipped until throttling is addressed?

Reply:
> AFAIK this only runs with the `/runIntegrationTests` command right? 

After a PR is merged, integ and e2e tests are run without `/runIntegrationTests`.

> And they aren't required to merge.

Yes, but a consistently red CI is not useful. Presumably when one runs `/runIntegrationTests` they want to see something that gives a useful signal of whether things are actually broken :)

Reply:
makes sense, will update